alexandria/storage/src/tests/list_test.cairo

```diff
@@ -14,6 +14,7 @@ trait IAListHolder<TContractState> {
         ref self: TContractState, index: u32, addrs_value: ContractAddress, numbers_value: u256
     );
     fn do_pop_front(ref self: TContractState) -> (Option<ContractAddress>, Option<u256>);
+    fn do_array(ref self: TContractState) -> (Array<ContractAddress>, Array<u256>);
```

Comment:
This (and related code) should be switched to `self: @TContractState` now as well for it to properly test `array`

Reply:
Woops, my bad